### PR TITLE
AK-16858 Update the documentation for connector-inet

### PR DESCRIPTION
**Pull request by spikewang**
Update the documentation for connector-inet and add examples.

### CODE REVIEW

**Comment by spikewang** [comment]
There are more options added and we will update the client to support those. This is just a quick catch-up of documentation to be more up-to-date.

File: alkira/resource_alkira_connector_internet.go [hunk 1]
```diff
@@ -6,48 +6,58 @@ import (
...
 	"github.com/alkiranet/alkira-client-go/alkira"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
 func resourceAlkiraConnectorInternet() *schema.Resource {
 	return &schema.Resource{
+		Description: "Manage Internet Exit.\n\n" +
+			"An internet exit is an exit from the CXP to the" +
+			"internet and allows and allows the traffic from" +
```
Comment (rvichery) [inline alkira/resource_alkira_connector_internet.go:16]:
```suggestion
			"internet and allows the traffic from" +
```

File: docs/resources/connector_internet.md [hunk 2]
```diff
@@ -3,34 +3,42 @@
...
+## Example Usage
 
+```terraform
+resource "alkira_connector_inet" "test1" {
+  name           = "test1"
+  cxp            = "US-WEST"
+  group          = "group1"
+  segment        = "seg1"
+}
+```
```
Comment (rvichery) [inline docs/resources/connector_internet.md:25]: I think `size` is missing and seems to be a required attribute per the schema.

File: examples/resources/alkira_connector_internet/resource.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+resource "alkira_connector_inet" "test1" {
+  name           = "test1"
+  cxp            = "US-WEST"
+  group          = "group1"
+  segment        = "seg1"
+}
+
```
Comment (rvichery) [inline examples/resources/alkira_connector_internet/resource.tf:7]: I think `size` is missing and seems to be a required attribute per the schema.

**Review by rvichery — CHANGES_REQUESTED** [review]
Couple of small changes required.

File: docs/resources/connector_internet.md (hunk 2 above)
Comment (spikewang) [reply docs/resources/connector_internet.md:25]: Good catch!

File: templates/guides/release-notes-v0.6.2.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Minor bug fix release.
+
+### resource `alkira_connector_aws_vpc`
+
+* Fix the update of connector
+
+* Fix the issues that `vpc_subnet` was specified but ignored.
+
+* Update documentation of the connector with more details and limiations.
```
Comment (rvichery) [inline templates/guides/release-notes-v0.6.2.md:16]:
```suggestion
* Update documentation of the connector with more details and limitations.
```

Comment (spikewang) [reply templates/guides/release-notes-v0.6.2.md]: 🤦

**Review by rvichery — APPROVED** [review]
👍

**Pull request merged**